app/views/search-for-your-gp.nunjucks

```diff
@@ -16,7 +24,7 @@
 </ul>
 <details class="" role="group">
   <summary data-analytics="summary" role="button" aria-controls="details-content-0" aria-expanded="false">
-    <span class="details__summary">If you haven't registered for online services</span>
+    <span class="details__summary" onmousedown="dcsMultiTrack('GPAppointment', 'Not_Registered', 'WT.dl', '121')">If you haven't registered for online services</span>
```

Comment:
yeah, that sounds fine. If @ajthomascouk fixes the docker image with the envs I'm happy to look at this separately